runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -9405,20 +9480,20 @@ static TR::Register* inlineCompareAndSwapObjectNative(TR::Node* node, TR::CodeGe
 
       generateLabelInstruction(LABEL, node, begLabel, cg);
 
-      generateRegMemInstruction(CMPRegMem(use64BitClasses), node, tmp, generateX86MemoryReference(cg->getVMThreadRegister(), offsetof(J9VMThread, evacuateBase), cg), cg);
+      generateRegMemInstruction(CMPRegMem(use64BitClasses), node, tmp, generateX86MemoryReference(cg->getVMThreadRegister(), offsetof(J9VMThread, evacuateBaseCompressed), cg), cg);
```

Comment:
Will this code work on 64-bit uncompressed or 32-bit ?

Reply:
Good point. Concurrent Scavenger is currently only supported on 64-bit compressedrefs. I've update this part to avoid confusion.